src/sonic-device-data/tests/media_checker

```diff
@@ -46,17 +49,32 @@ def check_media_dict(vendor_dict):
             print("Expecting settings for vendor type " + vendor_key)
             return False
 
+        lane_speed_hierarchy_exists = None
+
         for value_key in value_dict:
-            if value_key not in setting_keys:
-                print("Unknown media setting " + value_key)
-                return False
+            if value_key.startswith(lane_speed_key_prefix):
```

Comment:
@tshalvi please test a scenario where a port may have more than one SI param for SAME Speed